fresque/lib/alembic/env.py

```diff
@@ -1,17 +1,26 @@
-from __future__ import with_statement, unicode_literals, absolute_import, print_function
+from __future__ import with_statement
 from alembic import context
-from sqlalchemy import create_engine, pool
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
 
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
 
-if context.config.config_file_name is not None:
-    # Invoked by the "alembic" command, we can setup logging
-    from logging.config import fileConfig
-    fileConfig(context.config.config_file_name)
```

Comment:
Why remove this test? It's useful to check in which context the migration is run and not disrupt logging if it's not run from the command line.